projects/ngx-formentry/src/form-entry/form-renderer/form-renderer.component.scss

```diff
@@ -183,3 +183,19 @@ ng-select.form-control {
 .custom-text-area {
   width: 100%;
 }
+
+.container {
+  .cds--inline-notification--error {
+    display: flex;
+    align-items: center;
+
+    svg {
+      margin-left: 1rem;
+      margin-block-start: 0;
+    }
+  }
+
+  .cds--inline-notification--low-contrast::before {
```

Comment:
Here I've just removed the bottom border of the error container in the error pane that gets rendered after attempting to submit a form without filling all the mandatory fields.